doc/whatsnew/2.12.rst

```diff
@@ -198,3 +198,10 @@ Other Changes
 * Fix crash on ``open()`` calls when the ``mode`` argument is not a simple string.
 
   Partially closes #5321
+
+* Add ability to add ``end_line`` and ``end_column`` to the ``--msg-template`` option.
+  With the standard ``TextReporter`` this will add the line and column number of the
+  end of a node to the output of Pylint. If these numbers are unknown, they are represented
+  by an empty string.
+
+* Add ``end_line`` and ``end_column`` fields to the output of the ``JSONReporter``.
```

Comment:
It look like a breaking change that should wait for #4741. We have a 3.0 alpha branch, might be the time to dust it off ?

Reply:
Reverted! See https://github.com/PyCQA/pylint/issues/5380